roles/ceph-osd/tasks/docker/start_docker_osd.yml

```diff
@@ -20,7 +20,7 @@
 # starting the next task
 - name: prepare ceph osd disk
   shell: |   
-    docker run  --net=host \
+    docker run  --net=host --rm \
```

Comment:
It's been useful for me during development to leave this behind, so I can view logs if it fails.  I can also see that it's useful to make it go away, since this is a one-time action on a potentially long time machine.  I'm not sure what the solutions is, since I'm not convinced that adding *another* config knob is correct.  So I guess I'm asking: how useful do people find both situations: removing it and not removing it?